src/ui/setup/dynamic/devtools.ts

```diff
@@ -104,7 +116,12 @@ const SessionErrorMessages: Record<number, Partial<UnexpectedError>> = {
     content: "Our servers hiccuped but things should be back to normal soon.",
   },
   [SessionError.UnknownFatalError]: {
-    content: "Refreshing should help.\nIf not, please try recording again.",
+    content:
+      isWindows && isFirefox
+        ? "Our Windows browser is currently being upgraded to Chrome, which will reduce errors like these. Thanks for your patience!"
+        : "Refreshing should help.\nIf not, please try recording again.",
+    action: "refresh",
+    message: isWindows ? "Windows browser error" : "Browser error",
```

Comment:
I'd be more specific: `Windows replaying error`. Most people wont know what we mean by browser error, but replaying is clearer.